nautobot/extras/filters/__init__.py

```diff
@@ -905,6 +910,59 @@ class Meta:
         ]
 
 
+class JobQueueFilterSet(NautobotFilterSet, TenancyModelFilterSetMixin):
+    q = SearchFilter(
+        filter_predicates={
+            "name": "icontains",
+            "queue_type": "icontains",
+            "description": "icontains",
+            "tenant__name": "icontains",
+        },
+    )
+    tenant = NaturalKeyOrPKMultipleChoiceFilter(
+        queryset=Tenant.objects.all(),
+        label="Tenant (ID or name)",
+        to_field_name="name",
+    )
+    queue_type = django_filters.MultipleChoiceFilter(choices=JobQueueTypeChoices, null_value=None)
+
+    class Meta:
+        model = JobQueue
+        fields = [
+            "id",
+            "name",
+            "queue_type",
+            "description",
+            "tenant",
```

Comment:
IIRC you can omit `queue_type` and `tenant` here` since they're explicitly declared above?